tests/dummy/app/controllers/components-examples/flexberry-layerstree/settings-example.js

```diff
@@ -0,0 +1,542 @@
+import Ember from 'ember';
+import FlexberryLayersTreenodeActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-layerstreenode-actions-handler';
+
+export default Ember.Controller.extend(FlexberryLayersTreenodeActionsHandlerMixin, {
+  /**
+    Ember data store.
+
+    @property _store
+    @type DS.Store
+    @private
+  */
+  _store: Ember.inject.service('store'),
```

Comment:
As i know ember-data has initializer that injects store-service into all application routes, comtrollers & data-adapters.
So store must be available through this.get('store').